executor/set_test.go

```diff
@@ -137,6 +137,13 @@ func (s *testSuite2) TestSetVar(c *C) {
 	tk.MustExec("set character_set_results = NULL")
 	tk.MustQuery("select @@character_set_results").Check(testkit.Rows(""))
 
+	tk.MustExec("set @@session.ddl_slow_threshold=12345")
+	tk.MustQuery("select @@session.ddl_slow_threshold").Check(testkit.Rows("12345"))
+	c.Assert(variable.DDLSlowOprThreshold, 10, uint32(12345))
+	tk.MustExec("set session ddl_slow_threshold=\"54321\"")
+	tk.MustQuery("show variables like 'ddl_slow_threshold'").Check(testkit.Rows("ddl_slow_threshold 54321"))
+	c.Assert(variable.DDLSlowOprThreshold, 10, uint32(54321))
```

Comment:
should be `c.Assert(variable.DDLSlowOprThreshold, EQUALS, uint32(54321))`, Please take a look at https://labix.org/gocheck doc.

Reply:
The CI failed because of this.

Reply:
I am sorry for that.
I will fix it asap.